validator/main.go

```diff
@@ -56,5 +63,43 @@ func BaseToInterface(v *Base) IValidator {
 	return performer_factory(v)
 }
 
+func GetPostWithHeadlessBrowser(url string, regexp string) (post string, err error) {
+	headlessEntrypoint := lo.Sample(config.C.Headless.Urls)
+	headlessEntrypoint += "/v1/find"
+	request := headless.FindRequest{
+		Location: url,
+		Timeout:  "120s",
+		Match: headless.Match{
+			Type: "regexp",
+			MatchRegExp: &headless.MatchRegExp{
+				Selector: "*",
+				Value:    regexp,
+			},
+			MatchXPath: nil,
```

Comment:
Required to match both RegExp and XPath to avoid vulnerabilities?

Reply:
But `XPath` will change frequently depending on Elon's mood.